packer/files/etc/profile.d/workarea.sh

```diff
@@ -0,0 +1,27 @@
+#
+# Workarea Environment Configuration
+#
+
+# Change the following settings to match your external services
+# configuration if you are running your MongoDB, Elasticsearch, and/or
+# Redis servers on a different machine.
+#export WORKAREA_MONGOID_HOST="localhost:27015"
+#export WORKAREA_ELASTICSEARCH_URL="localhost:9200"
+#export WORKAREA_REDIS_HOST="localhost"
+#export WORKAREA_REDIS_PORT="6379"
+#export WORKAREA_REDIS_CACHE_HOST="localhost"
+#export WORKAREA_REDIS_CACHE_PORT="6379"
+
+# By default, apps on remote servers run in production mode. This
+# enables caching, host protection, and loads of other features that
+# help production apps run efficiently.
+export RAILS_ENV="production"
+
+# Configure your AWS S3 credentials to support uploading images. These
+# settings are **REQUIRED** or the application won't install properly.
+export WORKAREA_S3_ACCESS_KEY="CHANGE ME"
```

Comment:
this is a major footgun, the app REQUIRES you to ssh in prior to requesting a page in order to change this stuff. there's no way around it. is there any way we can just forego S3 config if the AWS keypair doesn't exist?